### PR TITLE
Observability - shopper insights and industry assets initial draft

### PR DESCRIPTION
**Pull request by dkirby-ms**

### CODE REVIEW

File: docs/azure_jumpstart_ag/contoso_hypermarket/observability/assets/_index.md [hunk 1]
```diff
@@ -1,8 +1,70 @@
...
+
+    ![A screenshot showing the maintenance worker dashboard](./maintenance_dashboard.png)
+
+>**Note:** The suggested prompts user interface widgets at the top of the screen are cosmetic only and aren't implemented in this release of Contoso Hypermarket.
+
+- Users can also interact with Contoso Hypermarket using natural language. Click on the [Jumpstart Cerebral](../../cerebral/) icon in the upper right to ask a question about the status of industrial assets.
+
+    ![A screenshot showing Jumpstart Cerebral location in the app](./cerebral_icon.png)
+
+    ![A screenshot showing Jumpstart Cerebral answering a question about industrial assets](./placeholder_until_it_works.png)
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/observability/assets/_index.md:36]: This screenshot is broken

File: docs/azure_jumpstart_ag/contoso_hypermarket/observability/assets/_index.md [hunk 2]
```diff
@@ -1,8 +1,70 @@
...
+
+    ![A screenshot showing the Industrial Assets Health menu of dashboards](./grafana_menu.png)
+
+- From this view, users can view real-time telemetry from industrial assets and filter by store and time dimension.
+
+    ![A screenshot showing the industrial assets Grafana dashboard](./grafana_dashboard.png)
+
+## Observability architecture components
+
+The Contoso Hypermarket observability architecture uses [Azure IoT Operations](https://learn.microsoft.com/azure/iot-operations/overview-iot-operations) to collect and send industrial assets telemetry using MQTT, an industry standard protocol. Assets including refrigerators, ovens, HVACs, and retail smart shelves send telemetry through the [[MQTT Broker](https://learn.microsoft.com/azure/iot-operations/manage-mqtt-broker/overview-iot-mq)] using an [edge-to-cloud data pipeline](../../data_pipeline/operational/_index.md).
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/observability/assets/_index.md:56]:
```suggestion
The Contoso Hypermarket observability architecture uses [Azure IoT Operations](https://learn.microsoft.com/azure/iot-operations/overview-iot-operations) to collect and send industrial assets telemetry using MQTT, an industry standard protocol. Assets including refrigerators, ovens, HVACs, and retail smart shelves send telemetry through the [MQTT Broker](https://learn.microsoft.com/azure/iot-operations/manage-mqtt-broker/overview-iot-mq)] using an [edge-to-cloud data pipeline](../../data_pipeline/operational/_index.md).
```

File: docs/azure_jumpstart_ag/contoso_hypermarket/observability/shopper_insights/_index.md [hunk 3]
```diff
@@ -5,4 +5,66 @@ title: Shopper insights
...
+
+    ![A screenshot showing the Industrial Assets Health menu of dashboards](./dashboard_menu.png)
+
+- From this view, users can view real-time telemetry from industrial assets and filter by store and time dimension.
+
+    ![A screenshot showing the industrial assets Grafana dashboard](./grafana_dashboard.png)
+
+## Observability architecture components
+
+The Contoso Hypermarket observability architecture uses [Azure IoT Operations](https://learn.microsoft.com/azure/iot-operations/overview-iot-operations) to collect and send industrial assets telemetry using MQTT, an industry standard protocol. Assets including refrigerators, ovens, HVACs, and retail smart shelves send telemetry through the [[MQTT Broker](https://learn.microsoft.com/azure/iot-operations/manage-mqtt-broker/overview-iot-mq)] using an [edge-to-cloud data pipeline](../../data_pipeline/operational/_index.md).
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/observability/shopper_insights/_index.md:56]:
```suggestion
The Contoso Hypermarket observability architecture uses [Azure IoT Operations](https://learn.microsoft.com/azure/iot-operations/overview-iot-operations) to collect and send industrial assets telemetry using MQTT, an industry standard protocol. Assets including refrigerators, ovens, HVACs, and retail smart shelves send telemetry through the [MQTT Broker](https://learn.microsoft.com/azure/iot-operations/manage-mqtt-broker/overview-iot-mq)] using an [edge-to-cloud data pipeline](../../data_pipeline/operational/_index.md).
```

File: docs/azure_jumpstart_ag/contoso_hypermarket/observability/assets/_index.md [hunk 4]
```diff
@@ -1,8 +1,70 @@
...
+- [**MQTT**](https://learn.microsoft.com/azure/iot-operations/manage-mqtt-broker/overview-iot-mq): A lightweight messaging protocol used for efficient communication between IoT devices and the cloud.
+- [**Prometheus**](https://prometheus.io/docs/introduction/overview/): An open-source monitoring and alerting toolkit used to collect and store metrics from industrial assets.
+- [**Grafana**](https://grafana.com/): A multi-platform open-source analytics and interactive visualization web application used to create dashboards and graphs from the metrics collected by Prometheus.
+- [**Azure Arc-enabled Kubernetes**](https://learn.microsoft.com/azure/azure-arc/kubernetes/overview): Manage and configure Kubernetes clusters running anywhere using Azure.
+
+![Screenshot of the architecture diagram for observability](./arch_diagram.png)
+
+## Next steps
+
+[Shopper insights](../shopper_insights/) - Learn how Contoso Hypermarket infuses AI to enhance retail store operations and shopper experience.
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/observability/assets/_index.md:68]: I think this section needs to be in bullets to have more structure

File: docs/azure_jumpstart_ag/contoso_hypermarket/observability/assets/_index.md (hunk 1 above)
Comment (dkirby-ms) [reply docs/azure_jumpstart_ag/contoso_hypermarket/observability/assets/_index.md:36]: waiting for fix to cerebral

**Pull request merged**